pkg/porter/create.go

```diff
@@ -9,35 +9,41 @@ import (
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+func (p *Porter) Create(bundleName string) error {
```

Comment:
Based off the decision above to keep current functionality, but add the ability to pass an argument and make a directory, we should separate out `Create()` `CreateinNewDir()` but try to copy as little code between the two as possible. As always, my names for functions aren't _great_ so feel free to use your own.

Reply:
Sure so basically how about adding a flag in create if it feels good say "--in -I"

Reply:
Or creating a separate function for the purpose

Reply:
I would think of it like helper function. I wouldn't add a flag, maybe a bool - so if they do `porter create` with an argument it sets the directory bool to true. If you want, we could discuss more in the community meeting too :)

Reply:
If you check for the existence of the argument when `porter create` is called, you can call `p.Create()` (which is cwd) if there is no argument. If it does exist then call `p.CreateInDir(dir string)`. 

You can reuse the logic in `p.Create()` only changing  where it’s written to so you don’t have a lot of duplicate logic.

Reply:
That sounds good, so let's go with the "porter create directory", does it sounds good.

Reply:
To clarify, I think what @troy0820  and I are talking about is just expanding `porter create` to accept `porter create [argument]` that argument being the directory name. Then if that argument is passed in you call `p.CreateInDir(dir string)`. Just re-clarifying because the way your comment there is worded makes it sound like you're making a command called `porter create directory` which is not exactly what I believe we're trying to head towards! :)

Reply:
Exactly, you are not creating another command, you are adding to the command that exists already (porter create).  

You will check the positional args (probably should use `func MaximumNArgs` as 1) and if there is one, you can create ~~change~~ the directory where it is going to be written with the arg passed in.  

That will create the directory and then you can reuse `p.Create()`.

Reply:
cc: @troy0820 @schristoff the required changes have been added. Kindly review.

Reply:
One minor nit on this, since this is an exported function can we add a comment describing the function at the top?
i.e 
```
//Create does things! (actually put what the function does here)
func (p *Porter) Create (bundleName string) error {
```